extensions/wikia/CommunityHeader/tests/NavigationTest.php

```diff
@@ -0,0 +1,377 @@
+<?php
+
+namespace CommunityHeader;
+
+use Wikia\Util\GlobalStateWrapper;
+
+class NavigationTest extends \WikiaBaseTest {
+	/**
+	 * @dataProvider exploreItemsProviders
+	 *
+	 * @param $globals
+	 * @param $expectedExploreItems
+	 */
+	public function testExploreItems( $globals, $expectedExploreItems ) {
+		$globals = new GlobalStateWrapper( [
+			'wgEnableCommunityPageExt' => $globals['wgEnableCommunityPageExt'],
+			'wgEnableDiscussions' => $globals['wgEnableDiscussions'],
+			'wgEnableForumExt' => $globals['wgEnableForumExt'],
+
+		] );
+
+		$result = $globals->wrap( function () {
+			$navigation = new Navigation();
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Immediately return this expression instead of assigning it to the temporary variable "$navigation". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1488)